keystrokes/src/components/Character/CharacterList.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react';
+import '../../stylesheets/character.css';
+import styleGuide from '../../stylesheets/style';
+import { ref } from '../../config/constants';
+import { List, ListItem, Typography, Divider } from '@material-ui/core';
+import CharacterCreationModal from './CharacterCreationModal';
+
+function CharacterList(props) {
+	const [characters, setCharacters] = useState({});
```

Comment:
Curious why you chose to use an object for the characters list?

Reply:
It seemed that firebase never gave you back an array if you passed one in, so i tried to keep the data consistent and have it as an object. Could be wrong about that though, didnt look very far into it.